docs/source/models/causes/neonatal/index.rst

```diff
@@ -280,7 +286,9 @@ For simplicity, we will not include YLDs in this model.
 Validation Criteria
 +++++++++++++++++++
 
-Neonatal mortality counts and rate in simulation should match GBD estimates.
+Neonatal deaths per live birth in simulation should be within 10% of same ratio derived from GBD.  (We don't expect it to match exactly because of (1) our interpolation of the RRs, and (2) we use a constant mortality hazard at each BW-GA level, rather than the GBD's more complex model.)
```

Comment:
No, actually, I think the overall deaths are the one thing we _do_ expect to match, since we're basing everything on ACMR (and the overall cause-specific deaths should match too since we're using GBD's CSMRs). What I _don't_ expect to match exactly is your second bullet point -- differential deaths by LBWSG category, for the reasons listed here.

Reply:
I will edit